polyhost/input/linux_kde_helper.py

```
@@ -70,4 +70,20 @@ def set_language(self, lang, country):
             return False, msg
 
     def get_current_language(self):
-        False, "Not Implemented" 
+        self.get_countries()
+        if not self.list:
+            return False, "No layout list loaded"
+        try:
+            result = subprocess.run(
+                ["qdbus", "org.kde.keyboard", "/Layouts", "getLayout"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)